test/fixtures/identities.yml

```diff
@@ -1,11 +0,0 @@
-# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
-
-one:
```

Comment:
Lots of old generated stuff in `/tests`. I already started a PR that introduces rspec, but will need some help with that.